sbt-plugin-test/referencedCrossProject/build.sbt

```diff
@@ -1,7 +1,7 @@
 lazy val referencedCrossProject = crossProject.
   crossType(CrossType.Pure).
   in(file(".")).
-  settings(scalaVersion := "2.11.7")
+  settings(scalaVersion := "2.11.8")
```

Comment:
Because builds on Jenkins are not isolated ... so this `referencedCrossProject` used a previously published scalajs-compiler_2.11.7 0.6.8-SNAPSHOT. But there was no previously published scalajs-compiler_2.11.7 for 0.6.8 (non snapshot).